src/admin/versions.js

```diff
@@ -20,7 +21,7 @@ function getLatestVersion(callback) {
 		headers['If-Modified-Since'] = versionCacheLastModified;
 	}
 
-	request('https://api.github.com/repos/NodeBB/NodeBB/tags', {
+	request(nconf.get('github_api') || 'https://api.github.com/repos/NodeBB/NodeBB/tags', {
```

Comment:
What I was suggesting was something like 

    (nconf.get('github_api') || 'https://api.github.com') + '/repos/NodeBB/NodeBB/tags'

Or, preferably, set the default nconf value for `github_api` and just do 

    nconf.get('github_api') + '/repos/NodeBB/NodeBB/tags'